src/dodal/devices/aperturescatterguard.py

```diff
@@ -38,6 +38,14 @@ class SingleAperturePosition:
         0, 0, 0, 0, 0
     )
 
+    def dict(self):
```

Comment:
Can we use the `dataclass.asdict()` method for this instead?

Reply:
yeah, I had actually already done that, just forgot to push :man_facepalming: